### PR TITLE
Fix doc

### PR DESCRIPTION
**Pull request by LanaTodorovic93**
Implements suggestions by @raoanirudh  

- [x] solve issues with formatting.
- [x] move secondary-perils.txt to underlying science
- [x] work on the corresponding input and output rst files

### CODE REVIEW

**Comment by CatalinaYepes** [comment]
Nice improvements @LanaTodorovic93 !

Below a few suggestions:

1. It will be nice to include the `Secondary perils` in the workflows list.
The liquefaction section has been moved the “underlying science”, as it should be, but still, we should reference it somewhere.
2. Since secondary peril `Underlying science` should be place in a new section for “Secondary perils”.
3. For the output section, it should be nice to indicate in which output file (the name of the output) the probability of liquefaction is stored.

**Comment by LanaTodorovic93** [comment]
Done

**Review by CatalinaYepes — APPROVED** [review]

**Review by raoanirudh — COMMENTED** [review]
Thank you @LanaTodorovic93 and @LanaTodorovic15!

File: doc/user-guide/inputs/secondary-perils-inputs.rst [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+
+Before demonstrating an example of a site model, we first discuss the implemented functions that are useful for 
+retrieving relevant inputs for sites.
+
+
+Getting raster values at sites
+------------------------------
+
+Digital elevation data and its derivatives are often given as rasters. However, in the case of probabilistic analysis 
+of secondary perils (particularly for risk analysis) the analyist may need to deal with sites that are not distributed
```
Comment (raoanirudh) [inline doc/user-guide/inputs/secondary-perils-inputs.rst:23]: analyist → analyst

File: doc/user-guide/inputs/secondary-perils-inputs.rst [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+
+Digital elevation data and its derivatives are often given as rasters. However, in the case of probabilistic analysis 
+of secondary perils (particularly for risk analysis) the analyist may need to deal with sites that are not distributed 
+according to a raster grid.
+
+Raster values may be extracted at sites using a GIS program to perform a spatial join, but following inconvenient 
+historical precedent, this operation often produces new data files instead of simply appending the raster values to the 
+point data file.
+
+Therefore we have implemented a simple function, `srap <https://github.com/gem/oq-engine/blob/ef33b5e0dfdca7a214dac99d4d7214086023ab39/openquake/sep/utils.py#L22>`_,
```
Comment (raoanirudh) [inline doc/user-guide/inputs/secondary-perils-inputs.rst:30]: `sample_raster_at_points(...) <https://github.com/gem/oq-engine/blob/engine-3.20/openquake/sep/utils.py#L19>`_

use full function name and link to a specific engine release

File: doc/user-guide/inputs/secondary-perils-inputs.rst [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+to get the raster values. This function requires the filename of the raster, and the longitudes and latitudes of the 
+sites, and returns a Numpy array with the raster values at each point. This function can be easily incorporated into 
+a Python script or workflow in this manner.
+
+Additional function for :math:`V_{s30}` estimates is implemented in the engine `here <https://github.com/gem/oq-engine/blob/ef33b5e0dfdca7a214dac99d4d7214086023ab39/openquake/sep/utils.py#L260>`_.
```
Comment (raoanirudh) [inline doc/user-guide/inputs/secondary-perils-inputs.rst:35]: Same as above, link to a specific engine release

File: doc/user-guide/inputs/secondary-perils-inputs.rst [hunk 4]
```diff
@@ -0,0 +1,70 @@
...
+Therefore we have implemented a simple function, `srap <https://github.com/gem/oq-engine/blob/ef33b5e0dfdca7a214dac99d4d7214086023ab39/openquake/sep/utils.py#L22>`_,
+to get the raster values. This function requires the filename of the raster, and the longitudes and latitudes of the 
+sites, and returns a Numpy array with the raster values at each point. This function can be easily incorporated into 
+a Python script or workflow in this manner.
+
+Additional function for :math:`V_{s30}` estimates is implemented in the engine `here <https://github.com/gem/oq-engine/blob/ef33b5e0dfdca7a214dac99d4d7214086023ab39/openquake/sep/utils.py#L260>`_. 
+It requires that the slope is calculated as the gradient :math:`\frac{dy}{dx}` rather than an angular unit, and the 
+study area is categorized as tectonically *active* or *stable*.
+
+We also provide a more general `wrapper function <https://github.com/gem/oq-engine/blob/ef33b5e0dfdca7a214dac99d4d7214086023ab39/openquake/sep/utils.py#L227>`_.
```
Comment (raoanirudh) [inline doc/user-guide/inputs/secondary-perils-inputs.rst:39]: Same as above, link to a specific engine release

**Review by raoanirudh — COMMENTED** [review]
One typo + minor comments

**Pull request merged**